### ISSUE TITLE
Fine-tune with custom-dataset

### ISSUE THREAD

**Issue by atharvjairath** [yangheng95/PyABSA#185]
Hi, 
I have a custom dataset in CSV format containing [ Text, Aspect, Label (Polarity) ]. I wanted to know how can I finetune any PyABSA model from your repository for APC.
Can you please list out the steps needed to achieve that?
I understand that I need to create a dataset in the required format to fine-tune, using [ABSA Dataset](https://github.com/yangheng95/ABSADatasets), but I don't see any method to directly convert the data from CSV format to the required one i.e [this](https://github.com/yangheng95/ABSADatasets/tree/v1.2/datasets/apc_datasets/100.CustomDataset)

In addition to this, do you also have a script to store inference in CSV directly instead of printing?
Thanks in advance!

**Comment by yangheng95** [comment]
Currently, the datasets are either processed by users-selves or annotated using the DPT tool in ABDADatasets, no available scripts now for converting, as I cant foresee the CSV format of other users.

To use APC models, you can: 
- Write a script to convert your data and put it in the same location of ABSADatasets(named integrated_datasets in your project)
- Select a demo script for APC, e.g., [here](https://github.com/yangheng95/PyABSA/blob/release/demos/aspect_polarity_classification/train_apc_english.py)

It already have a saving function, you can just activate saving by setting `save_result=True` as this [example](https://github.com/yangheng95/PyABSA/blob/3aff0726bb1f03421f94272bf1b20fb5b8cbb8e1/demos/aspect_polarity_classification/sentiment_inference.py#L42)

**Comment by atharvjairath** [comment]
@yangheng95 Thank you for such a detailed reply, I will write the script for the same and if possible will contribute.
